index.html

```diff
@@ -248,17 +316,23 @@ <h1>Get started</h1>
                 <label for="email">Your email</label>
                 <input
                   type="email"
+                  onchange="storeData();"
                   id="email"
                   name="email"
                   placeholder="name@email.com"
                   required
+                  pattern="[a-z@.]+"
                 />
               </li>
             </ul>
```

Comment:
- [ ] Kindly make sure the error message is displayed is shown close to the submit button yours is below the email field.
![image](https://user-images.githubusercontent.com/61408860/168234907-6f85e7a5-e935-451c-a3cb-a7f1142d82a1.png)